ksp_plugin_test/pile_up_test.cpp

```diff
@@ -596,6 +597,50 @@ TEST_F(PileUpTest, Serialization) {
   EXPECT_THAT(message, EqualsProto(second_message));
 }
 
+TEST_F(PileUpTest, SerializationCompatibility) {
+  MockEphemeris<Barycentric> ephemeris;
+  p1_.increment_intrinsic_force(
+      Vector<Force, Barycentric>({1 * Newton, 2 * Newton, 3 * Newton}));
+  p2_.increment_intrinsic_force(
+      Vector<Force, Barycentric>({11 * Newton, 21 * Newton, 31 * Newton}));
+  TestablePileUp pile_up({&p1_, &p2_},
+                         astronomy::J2000,
+                         DefaultProlongationParameters(),
+                         DefaultHistoryParameters(),
+                         &ephemeris);
+
+  serialization::PileUp message;
+  pile_up.WriteToMessage(&message);
+
+  // Clear the children to simulate pre-Cesàro serialization.
+  message.mutable_history()->clear_children();
```

Comment:
Check that `message.history().timeline_size()` is 1 so that it is clear which code path is being tested.